api/apps/oura/get.ts

```diff
@@ -0,0 +1,34 @@
+import createAPIGatewayProxyHandler from "package/backend/createAPIGatewayProxyHandler";
+
+// https://cloud.ouraring.com/v2/docs
+
+type ParamProps = {
+  dataType: string;
+  date: string;
+  startDate: string;
+  endDate: string;
+  token: string;
+};
+const logic = async ({
+  dataType,
+  date, // TODO - need more data
+  endDate,
+  startDate,
+  token,
+}: ParamProps) => {
+  const url = `https://api.ouraring.com/v2/usercollection/${dataType}?start_date=${startDate}&end_date=${endDate}`;
+  const headers = { Authorization: `Bearer ${token}` };
+
+  try {
+    const response = await fetch(url, { method: "GET", headers });
+    const data = await response.json();
+    return { data };
+  } catch (error) {
+    return { error };
```

Comment:
we probably want to just point to error.message.

also, this won't be an `error` from the API request. It will likely be either:
- the actual `fetch` call failing
- if the json parse failed

To get the API response error, we'll want to check if `response.ok` is false, then return the data to an error key there as well